contrib/build_rpm.sh

```diff
@@ -70,3 +69,16 @@ if [ -d ~/rpmbuild/BUILD ]; then
 fi
 
 rpmbuild --rebuild ${extra_arg:-} podman-*.src.rpm
+createrepo -v ~/rpmbuild/RPMS/x86_64
```

Comment:
For consistency with line 75, could we make this /${HOME}/rpmbuild/RPMS/x86_64 ?

Reply:
IIRC, rpmbuild checks an env. var. for the preferred location.  The script should probably reference that instead of hard-coding.  Otherwise this build could break in unexpected ways.